chore(release): 5.0.2-alpha.0 (#9601)
OpenTrons/opentrons-api: api/release-notes.md

 - The most recent protocol analysis and run are stored on the robot to enable easy, quick re-running via the Opentrons App
 - A new way to use labware offsets in Jupyter notebook and SSH sessions

## 5.0.1 to 5.0.2

> The 5.0.2 hotfix release contains two changes. The first allows clients to POST individual labware definitions to an run resource. This fix is necessary for a front end fix that allows python protocols that load custom labware inline to still take advantage of the new Labware Position check flow. The Second fix passes user defined labware labels through via load labware command parameters. This second fix is necessary to allow users to inspect these custom labels while viewing the Labware Setup step in the Opentrons App.

Reviewer: I think this might be too technically detailed for end users. Let's pare it way down

```suggestion
The 5.0.2 hotfix release contains two fixes in the robot software:

1. The robot now understands how to use labware loaded via `load_labware_from_definition` during Labware Position Check.
2. User-defined labware labels are now returned to the Opentrons App from protocol analyses and runs.
```

## 5.0.0 to 5.0.1

The 5.0.1 hotfix release does not contain any changes to the robot software

## Known Issues

- Your last run protocol will not be saved through robot reboots.

> - The expected mapping of protocol load order to usb port info, in the case of multiple instances of a single module type in a protocol, is inconsistent.

Reviewer: ```suggestion
- When you load multiple instances of the same module type in a protocol, the USB port numbers shown in "Module Setup" are incorrect.
```
